components/locale-provider/nb_NO.tsx

```diff
@@ -0,0 +1,60 @@
+import moment from 'moment';
+moment.locale('nb');
+
+// import Pagination from 'rc-pagination/lib/locale/nb_NO';
+import DatePicker from '../date-picker/locale/nb_NO';
+import TimePicker from '../time-picker/locale/nb_NO';
+import Calendar from '../calendar/locale/nb_NO';
+
+export default {
+  locale: 'nb',
+  DatePicker,
+  TimePicker,
+  Calendar,
+  // Replace with rc-pagination/lib/locale/nb_NO
+  // when rc-pagination supports nb_NO
+  Pagination: {
```

Comment:
Done. [Link to PR](https://github.com/react-component/pagination/pull/103). As soon as the Pagination's PR is merged I will update this PR.